README.md

```diff
@@ -1,12 +1,14 @@
+# Rust Secp256k1
+
 [![Build Status](https://travis-ci.org/rust-bitcoin/rust-secp256k1.png?branch=master)](https://travis-ci.org/rust-bitcoin/rust-secp256k1)
 
 [Full documentation](https://docs.rs/secp256k1/)
 
-## rust-secp256k1
-
 `rust-secp256k1` is a wrapper around [libsecp256k1](https://github.com/bitcoin-core/secp256k1),
-a C library by Pieter Wuille for producing ECDSA signatures using the SECG curve
-`secp256k1`. This library
+a C library by Pieter Wuille for producing ECDSA signatures using the [SECG](https://www.secg.org/)
```

Comment:
"for producing ECDSA signatures" sounds weird since it can also verify them and also produce/verify schnorr signatures and also perform ECDH and also tweak the keys and probably something I forgot.

"a C library implementing various cryptographic functions using the SECG curve secp256k1" might be good.

Reply:
Used as suggested, note this removes Peter's  name - this seems correct since it does not appear in the readme of libsecp256k1 (I thought I remembered seeing it there before but not sure).

cc @sipa in case he would like the attribution to remain.

Reply:
We could say "created by Pieter Wuille". But I agree that we should follow the README, which presumably attributes it to "the bitcoin core developers" or something.

Reply:
Oh yeah, I didn't mean to drop attribution  but it's a good question which attribution is the correct one.